<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kontakt</class>
 <widget class="QWidget" name="Kontakt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="lista_osob">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>311</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDodaj">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEdytuj">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Edytuj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtoPodglad">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Podglad</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSkasuj">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Usun</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sort_Imie">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt; Imie:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sort_Nazwisko">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt; Nazwisko:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sort_firma">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt; Firma:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
